bitnami/mariadb/Chart.yaml

```
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 10.5.12
+appVersion: 10.6.0
```

```suggestion
appVersion: 10.5.12
```
You should change the `version` field instead of the `appVersion` one:
```diff
-version: 9.4.4
+version: 9.5.0
```